Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs

```diff
@@ -621,10 +771,22 @@ public void onInviteFailed(String message)
 
 		public static Action<Provider, string> OnInviteCancelled = delegate {};
 
+		public static Action<Provider, bool, string> OnGetLeaderboardsStarted = delegate {};
```

Comment:
@EugeneButusov that's what I wanted, but @Blinkq claims that this cannot be done for delegate methods (I looked it up a bit and it seems correct).